cypress/support/api/requests/Product.js

```diff
@@ -148,7 +148,7 @@ export function createVariant({
   productId,
   sku,
   warehouseId,
-  quantityInWarehouse,
+  quantityInWarehouse = 1,
```

Comment:
Why we are starting with 1?

Reply:
In most cases we need variant with quantity 1